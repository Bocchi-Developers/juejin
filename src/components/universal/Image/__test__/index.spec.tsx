import { describe, expect, it } from 'vitest'

import { render } from '@testing-library/react'

import mountTest from '~/test/mountTest'

import { Image } from '..'

mountTest(Image)

describe('Card', () => {
  it('snapshot', () => {
    const markdown = render(<Image className={'test'} />)
    expect(markdown.container).matchSnapshot()
  })
})
