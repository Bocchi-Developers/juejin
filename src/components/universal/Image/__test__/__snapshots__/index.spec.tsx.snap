// Vitest Snapshot v1

exports[`Card > snapshot 1`] = `
<div>
  <div
    class="arco-image arco-image-loading test"
    style="opacity: 1;"
  >
    <img
      class="arco-image-img"
      src="undefined"
      tabindex="0"
    />
    <div
      class="arco-image-overlay"
    />
  </div>
</div>
`;
